patch/armhf_secondary_boot_online.patch

```diff
@@ -0,0 +1,56 @@
+From 500d48948f43d18a07894b5fd4f891afbade1d79 Mon Sep 17 00:00:00 2001
+From: Antony Rheneus <arheneus@marvell.com>
+Date: Thu, 1 Jul 2021 20:34:01 +0530
+Subject: [PATCH 1/1] From 28c199cfda1746644211893b6614acbac970515b Mon Sep 17
```

Comment:
Is that commit in upstream Linux master? I didn’t see it.

Reply:
they are going to upstream it. the model is armhf.

Reply:
armada family.